homework/module_1-hw_standardlib/task_3.py

```diff
@@ -0,0 +1,20 @@
+import os
+import csv
+import json
+
+
+def converter_to_json(file_path: str, converted_file_name: str = "converted") -> None:
+    data_list = []
+    # deserializing csv into python object (reading data from the file and writing into data_list)
+    with open(file_path, newline="") as file:
+        reader = csv.DictReader(file)
+        for row in reader:
```

Comment:
You can use `data_list = list(reader)` here

Reply:
fixed